bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -264,9 +298,9 @@ public Response startDialog(
                 return JSONResponse.createErrorResponse(Status.NOT_FOUND, "Text-to-Speech not found");
             }
         }
-        HumanLanguageInterpreter hli = null;
+        Collection<HumanLanguageInterpreter> hli = null;
         if (hliId != null) {
-            hli = voiceManager.getHLI(hliId);
```

Comment:
Please update the parameter description, rename it into `hliIds` and rename your local variable into `hlis` or `interpreters`.